grafana-dashboard-sync/export-and-commit

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
During my first approach I used external library for exporting grafina dashboards to files and the bash script below to commit changes. Unfortunately the external library was cutting units from exported dashboards so I decided to rewrite exporting part in golang.

So the final approach is the following: We export dashboards using go binary and we commit using bash script.
There is alternative approach though: We could use https://github.com/go-git/go-git library which has ability to clone repository in-memory so we can potentially do all the stuff within single binary without polluting the original fs. Unfortunately I was lacking of time to implement single binary solution.